pkg/netceptor/conn_simple_test.go

```diff
@@ -0,0 +1,185 @@
+package netceptor
```

Comment:
Im not quite sure why there is a need for both a conn_simple_test as well as a con_tls_test, I would expect these to be combined into one file 😄